src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -1414,61 +1414,53 @@
     </Style>
 
     <Style x:Key="SNodeTextButton" TargetType="{x:Type Button}">
+        <Setter Property="FontSize" Value="17px"/>
+        <Setter Property="Foreground" Value="{StaticResource Blue300Brush}" />
+        <Setter Property="Background" Value="#4D4D4D" />
```

Comment:
is this color reused in a lot of places? Can it be pulled out?

Reply:
Looks like it's used in 5 places in DynamoModern. It already exists in ColorsAndBrushes, but for more specific uses. I wasn't sure how far to take a style revision, so I defaulted to just making the changes necessary to use the styles for the new component. From my interaction so far it looks like there are a _lot_ of optimization opportunities like this: unnecessary styles, duplicated sections, hard-coded colors.